doc/tutorials/bare_flame_game.md

```diff
@@ -0,0 +1,186 @@
+you have basic familiarity with using the command line. Android Studio is not
+a strict requirement, you can use other IDEs too, such as [Visual Studio Code].
+
+
+## 1. Check flutter installation
+
+First, let's verify that your Flutter SDK was installed correctly and is
+accessible from the command line:
+
+```console
+$ flutter doctor
+[✓] Flutter (Channel stable, 2.8.0, on macOS 12.2 21D49 darwin-x64, locale en)
+[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
+[✓] Xcode - develop for iOS and macOS (Xcode 13.1)
+[✓] Chrome - develop for the web
+[✓] Android Studio (version 2020.3)
+[✓] Connected device (1 available)
+```
+
+Your output will be slightly different, but the important thing is to verify
+that no errors are reported, and that your Flutter version is **2.5.0** or 
+above.
+
+
+## 2. Create project directory
+
+Now you need to come up with a name for your project. The name can only use
+lowercase latin letters, digits, and underscores. It must also be a valid Dart
+identifier (thus, for example, it cannot be a keyword).
+
+In this tutorial I'm going to call my project **syzygy**, which is a totally
+real not-made-up word.
+
+Create the directory for your new project:
+
+```console
+$ mkdir -p ~/projects/syzygy
+$ cd ~/projects/syzygy
+```
+
+
+## 3. Initialize empty Flutter project
+
+In order to turn this barren directory into an actual Flutter project, run the
+following command:
+
+```console
+$ flutter create .
+```
+(I have omitted the output for brevity, but there will be lots of output).
+
+You can verify that the project files were created successfully:
+```console
+$ ls 
+README.md               android/   lib/           pubspec.yaml   test/
+analysis_options.yaml   ios/       pubspec.lock   syzygy.iml     web/
+```
+
+
+## 4. Open the project in Android Studio
+
+Launch Android Studio, then in the project selection window choose `[Open]`
+and navigate to your project directory. With any luck, the project will now
+look like this:
+
+![](../images/tutorials/android-studio-screenshot-1.webp)
+
+If you see only the `main.dart` file but not the side panel, then click the
+vertical `[Project]` button at the left edge of the window.
+
+Before we proceed, let's fix the view in the left panel. Locate the button
+in the top left corner that says `[Android]` in the screenshot. In this 
+dropdown select the first option "Project". Your project window should now
+look like this:
+
+![](../images/tutorials/android-studio-screenshot-2.webp)
+
+The important part is that you should be able to see all files in your 
+project directory.
+
+
+## 5. Clean up the project files
+
+The default project created by Flutter is not very useful for making a Flame
+game, so we should get rid of it.
+
+First, open the file `pubspec.yaml` and replace it with the following code
+(adjusting the `name` and `description` to match your project):
+
+```yaml
+name: syzygy
+description: Syzygy flame game
+version: 0.0.0
```

Comment:
0.1.0

Reply:
`0.0.0` is symbolic of this being the "zero state" project. But ultimately it's up to the developer to decide on how to do versioning for their game.

Reply:
I thought `0.1.0` was the default from `flutter create`, but it's `1.0.0+1`, I don't see a reason to change from the default when it is a game and not a library?

Reply:
To me, `1.0.0` sounds like the first version of the product. So, if the intended product is the counting app (as `flutter create` makes), then it's perhaps reasonable to call it version 1. But if the intended product is a Flame game, then we need to remove the parts that make the counting app, and start from a clean slate. That clean slate is not a finished product, not even a started product, so I think it deserves version `0.0.0`. But then of course the developer is free to choose their own versioning scheme.

This version string will need to be updated when it's time to publish the game, but until then I feel designating it as `1.0.0+1` is a bit inappropriate.

It would be great to have a separate tutorial for the publishing process, which could go over these questions in more detail.

Reply:
It creates it with the same version when you run it with `flutter create . --template=skeleton` (which definitely is not something done), but I agree, it is a strange version to start on so let's go with `0.0.0`.

> It would be great to have a separate tutorial for the publishing process, which could go over these questions in more detail.

A publishing tutorial is probably something that would fit better in the docs than on the tutorials page too.